src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -191,5 +194,9 @@ class Microsoft::Terminal::Core::Terminal final :
     std::vector<SMALL_RECT> _GetSelectionRects() const;
     const SHORT _ExpandWideGlyphSelectionLeft(const SHORT xPos, const SHORT yPos) const;
     const SHORT _ExpandWideGlyphSelectionRight(const SHORT xPos, const SHORT yPos) const;
+    void _ExpandDoubleClickSelectionLeft(const COORD position);
+    void _ExpandDoubleClickSelectionRight(const COORD position);
+    const bool _DoubleClickDelimiterCheck(std::wstring_view cellChar) const;

```

Comment:
nit: rename this to `_isWordDelimiter` or something:

1. it returns a boolean, so 'is' is the appropriate modality; using the noun "check" is less ideal
2. its use could be expanded